packages/fiori/src/DynamicSideContent.js

```diff
@@ -0,0 +1,572 @@
+		 *
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		"default": {
+			type: HTMLElement,
+		},
+
+		/**
+		 * Defines the Side Content.
+		 *
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		 "sideContent": {
+			type: HTMLElement,
+		},
+
+	},
+	events: /** @lends sap.ui.webcomponents.fiori.DynamicSideContent.prototype */ {
+
+		/**
+		 * Fires when the current breakpoint has been changed.
+		 * @event sap.ui.webcomponents.fiori.DynamicSideContent#layout-change
+		 * @param {String} currentBreakpoint the current breakpoint.
+		 * @param {String} previousBreakpoint the breakpoint that was active before change to current breakpoint.
+		 * @param {Boolean} mainContentVisible visibility of the main content.
+		 * @param {Boolean} sideContentVisible visibility of the side content.
+		 * @public
+		 */
+		"layout-change": {
+			details: {
+				currentBreakpoint: {
+					type: String,
+				},
+				previousBreakpoint: {
+					type: String,
+				},
+				mainContentVisible: {
+					type: Boolean,
+				},
+				sideContentVisible: {
+					type: Boolean,
+				},
+			},
+		},
+
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * <code>ui5-dynamic-side-content</code> is a layout component that allows additional content
+ * to be displayed in a way that flexibly adapts to different screen sizes. The side
+ * content appears in a container next to or directly below the main content
+ * (it doesn't overlay). When the side content is triggered, the main content becomes
+ * narrower (if appearing side-by-side). The side content contains a separate scrollbar
+ * when appearing next to the main content.
+ *
+ *
+ * <h3>Usage</h3>
+ *
+ * <i>When to use?</i>
+ *
+ * Use this component if you want to display relevant information that is not critical
+ * for users to complete a task. Users should have access to all the key functions and
+ * critical information in the app even if they do not see the side content. This is
+ * important because on smaller screen sizes it may be difficult to display the side
+ * content in a way that is easily accessible for the user.
+ *
+ * <i>When not to use?</i>
+ *
+ * Don't use it if you want to display navigation or critical information that prevents
+ * users from completing a task when they have no access to the side content.
+ *
+ *
+ * <h3>Responsive Behavior</h3>
+ *
+ * Screen width > 1440px
+ *
+ * <ul><li>Main vs. side content ratio is 75 vs. 25 percent (with a minimum of 320px
+ * each).</li>
+ * <li>If the application defines a trigger, the side content can be hidden.</li></ul>
+ *
+ * Screen width <= 1440px and > 1024px
+ *
+ * <ul><li>Main vs. side content ratio is 66.666 vs. 33.333 percent (with a minimum of
+ * 320px each). If the side content width falls below 320 px, it automatically slides
+ * under the main content, unless the app development team specifies that it should
+ * disappear.</li></ul>
+ *
+ * Screen width <= 1024px and > 720px
+ *
```

Comment:
I suggest to reorder the sentence like:
<ul><li>If the <code>sideContentFallDown</code> is not set to <code>OnMinimumWidth</code>, the side content ratio is fixed to 340px, and the main content takes the rest of the width.
If the <code>sideContentFallDown</code> is set to <code>OnMinimumWidth</code>, the above ratios are valid only for screen width <= 1024px and > 960px, but for screen width <= 960px and > 720px the side content falls below the main content.
 </li></ul>

or better: 
<ul><li>
The side content ratio is fixed to 340px, and the main content takes the rest of the width. Only if the <code>sideContentFallDown</code> is set to <code>OnMinimumWidth</code> and screen width is <= 960px and > 720px the side content falls below the main content.
</li></ul>

Reply:
Second is more clear.